canonical/ght
# Update Github actions and add unit tests (WD-467)

## Done
- Added unit tests.
- Modified Github actions to run only unit tests.
## QA
- Check that all tests pass by running `npm run test`.
- Check that Github actions is properly configured.
